src/js/components/Block/Container.tsx

```diff
@@ -112,7 +90,9 @@ const _Container = React.forwardRef(({ children, isDragging, isSelected, isOpen,
         ".block-body > .block-toggle:focus": {
           opacity: 1
         },
-        "&.is-hovered-not-child > .block-body > .block-toggle, &:focus-within > .block-body > .block-toggle": { opacity: "1" },
+        "&.is-hovered-not-child > .block-body > .block-toggle, &.is-editing > .block-body > .block-toggle": {
+          opacity: "1"
+        },
```

Comment:
Fixes block toggles appearing when any child was focused. Now only the block being edited, or any hovered block should show the toggle.